ts/output/common/Wrapper.ts

```diff
@@ -1177,6 +1177,16 @@ export class CommonWrapper<
     if (scale === null) {
       scale = this.bbox.scale;
     }
+    const t = this.font.params.rule_thickness;
+    if (length === 'medium') {
+      return t;
+    }
+    if (length === 'thin') {
+      return (2 / 3) * t;
+    }
+    if (length === 'thick') {
+      return (5 / 3) * t;
+    }
```

Comment:
Can you get the coefficient from a mapping of length to number and then  
`return coeff ? coeff * t : LENGTHS....` ?
While that means we get an extra multiplication on `length === 'medium'`, we avoid the linear complexity comparison.

Reply:
OK, done.  Please check that this is OK.